### PR TITLE
Change "see issue #5613" to "see details"

### PR DESCRIPTION
**Pull request by silkeholmebonnen**
## Issue

[AVO-171](https://linear.app/electricitymaps/issue/AVO-171/change-see-issue-5613-to-see-more-details)

## Description

This PR changes the outage message so it says "see details" instead of "see issue #5613". 
I thought "see details" was more correct than "see more details", but I can change it if you don't agree.

### Preview

This picture is a mock since there are no outages at the moment (as far as I can tell).
image

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/features/panels/zone/EstimationCard.tsx [hunk 1]
```diff
@@ -241,17 +241,17 @@ function OutageMessage({
...
       {truncateString(outageData.message, 300)}{' '}
       {outageData?.issue && outageData.issue != 'None' && (
         <span className="mt-1 inline-flex">
-          See{' '}
           <a
             className="inline-flex text-sm font-semibold text-black underline dark:text-white"
             href={`https://github.com/electricitymaps/electricitymaps-contrib/issues/${outageData.issue}`}
           >
-            <span className="pl-1 underline">issue #{outageData.issue}</span>
+            <span className="pl-1 underline">{t('estimation-card.outage-details')}</span>
```
Comment (VIKTORVAV99) [inline web/src/features/panels/zone/EstimationCard.tsx:254]: Nice to get the whole string translatable as well!

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM!

**Comment by madsnedergaard** [comment]
Thanks for fixing it 👏

**Comment by silkeholmebonnen** [comment]
> Looks good, might be nice to add a German translation here as they're probably the most affected by outages

Wouldn't it be weird if the message is in english and then it says see details in German?

**Pull request merged**